exporter/metrics_exporter.go

```diff
@@ -85,7 +85,7 @@ func (m *metricsExporter) recordMetrics() {
 		&meta.ListOptions{},
 	)
 	if err != nil {
-		log.Fatal(err) // TODO: Let's not crash every time there's an error
+		log.Println(err) // TODO: Is this the best way to deal with errors?
 	}
 	m.totalProjects.Set(float64(len(projects.Items) +
 		int(projects.RemainingItemCount)))
```

Comment:
This should occur conditionally when there was no error, otherwise, you're setting a zero as the project count.

This guidance applies in several other places as well.

Reply:
Fixed. For the job count, since setting the gauge happens outside of the scope of the error, I created a separate variable to track errors that happen within the job counting process. Is that okay?